src/widgets/media-requests/MediaRequestStatsTile.tsx

```diff
@@ -1,4 +1,4 @@
-import { Card, Center, Flex, Stack, Text } from '@mantine/core';
+import { Box, Card, createStyles, Flex, FlexProps, Stack, Text } from '@mantine/core';
```

Comment:
```suggestion
import { Card, createStyles, Flex, Stack, Text } from '@mantine/core';
```
FlexProps and Box are unused, right?